independent-publisher-connectors/OMDb API/apiDefinition.swagger.json

```diff
@@ -0,0 +1,167 @@
+          },
+          {
+            "name": "i",
+            "in": "query",
+            "description": "A valid IMDb ID (e.g. tt1285016)",
+            "required": false,
+            "type": "string",
+            "x-ms-summary": "ID to be used for searching the database",
+            "x-ms-visibility": "important"
+          },
+          {
+            "name": "y",
+            "in": "query",
+            "description": "Year of release",
+            "required": false,
+            "type": "integer",
+            "x-ms-summary": "To refine the search by finding movies/series from a particular year",
+            "format": "int32"
+          },
+          {
+            "name": "type",
+            "in": "query",
+            "description": "Return movie or series or episode",
+            "required": false,
+            "type": "string",
+            "enum": ["movie", "series", "episode"],
+            "x-ms-summary": "Select the type to be returned"
+          },
+          {
+            "name": "page",
+            "in": "query",
+            "description": "Page number to return",
+            "required": false,
+            "type": "integer",
+            "default": 1,
+            "format": "int32",
+            "x-ms-summary": "Select the page number. Ranges from 1-100"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successful operation",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Search": {
+                  "type": "array",
+                  "items": {
+                    "type": "object",
+                    "properties": {
+                      "Title": {
+                        "type": "string",
+                        "description": "Title"
+                      },
+                      "Year": {
+                        "type": "string",
+                        "description": "Year"
+                      },
+                      "imdbID": {
+                        "type": "string",
+                        "description": "imdbID"
+                      },
+                      "Type": {
+                        "type": "string",
+                        "description": "Type"
+                      },
+                      "Poster": {
+                        "type": "string",
+                        "description": "Poster"
+                      }
+                    }
+                  },
+                  "description": "Search"
+                }
+              }
+            }
+          },
+          "401": {
+            "description": "Not authenticated"
+          }
+        },
+        "description": "Based on the title/ID/Keyword provided searches the database for matching results and shows all the available results. Further query parameters may be used to refine the result like Year of release, Movie or Series, the response type to return and Page number",
```

Comment:
all Operations descriptions should end with a period. No period required only if there's a single word.
Please see this useful [documentation](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#operations)
We suggest you shorten this description so it can all show in the box.

Reply:
Done